React/nested-event-handling.md

```diff
@@ -134,51 +175,80 @@ There are a few things to consider when making this change to our application. T
 
 If you answered NO to question #3, you're on to something big. The data related to the students is tracked in the `StudentCollection` while the button to mark an individual student "present" should really be on each individual `Student` component. Our challenge then is to use the tools we have been given to _propagate_ the button press event from one component to another.
 
-#### The button
+#### The Button
 Let's start by updating the `render` function of the `Student` component to include the `isPresent` property, and a button.
 
 ```js
-<p>
```

Comment:
Would it make sense to change this to not reference the `render` function, since `Student` is no longer a class?